packages/components/src/Tooltip/src/Tooltip.module.css

```diff
@@ -0,0 +1,67 @@
+.hop-Tooltip {
+    --hop-Tooltip-max-inline-size: 25rem;
+    --hop-Tooltip-slide-amount: 0.25rem; /* Tokens not available here */
+
```

Comment:
Could we test a custom trigger?

Reply:
like just plain text for example?

Reply:
https://wl-orbiter-website.netlify.app/?path=/docs/tooltip--default-story#custom-trigger you can get inspiration from here